internal/service/verify/resource_voice_phrase.go

```diff
@@ -0,0 +1,371 @@
+		return
+	}
+
+	// Read Terraform plan data into the model
+	resp.Diagnostics.Append(req.Plan.Get(ctx, &plan)...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	// Build the model for the API
+	VoicePhrase, d := plan.expand()
+	resp.Diagnostics.Append(d...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	// Run the API call
+	var response *verify.VoicePhrase
+	resp.Diagnostics.Append(framework.ParseResponse(
+		ctx,
+
+		func() (any, *http.Response, error) {
+			return r.client.VoicePhrasesApi.UpdateVoicePhrase(ctx, plan.EnvironmentId.ValueString(), plan.Id.ValueString()).VoicePhrase(*VoicePhrase).Execute()
+		},
+		"UpdateVoicePhrase",
+		framework.DefaultCustomError,
+		sdk.DefaultCreateReadRetryable,
+		&response,
+	)...)
+
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	// Update the state to save
+	state = plan
+
+	// Save updated data into Terraform state
+	resp.Diagnostics.Append(state.toState(response)...)
+	resp.Diagnostics.Append(resp.State.Set(ctx, state)...)
+}
+
+func (r *VoicePhraseResource) Delete(ctx context.Context, req resource.DeleteRequest, resp *resource.DeleteResponse) {
+	var data *voicePhraseResourceModel
+
+	if r.client == nil {
+		resp.Diagnostics.AddError(
+			"Client not initialized",
+			"Expected the PingOne client, got nil.  Please report this issue to the provider maintainers.")
+		return
+	}
+
+	// Read Terraform prior state data into the model
+	resp.Diagnostics.Append(req.State.Get(ctx, &data)...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	// Run the API call
+	resp.Diagnostics.Append(framework.ParseResponse(
+		ctx,
+
+		func() (any, *http.Response, error) {
+			r, err := r.client.VoicePhrasesApi.DeleteVoicePhrase(ctx, data.EnvironmentId.ValueString(), data.Id.ValueString()).Execute()
+			return nil, r, err
+		},
+		"DeleteVoicePhrase",
+		framework.CustomErrorResourceNotFoundWarning,
+		sdk.DefaultCreateReadRetryable,
+		nil,
+	)...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+}
+
+func (r *VoicePhraseResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
+	splitLength := 2
+	attributes := strings.SplitN(req.ID, "/", splitLength)
+
+	if len(attributes) != splitLength {
+		resp.Diagnostics.AddError(
+			"Unexpected Import Identifier",
+			fmt.Sprintf("invalid id (\"%s\") specified, should be in format \"environment_id/voice_phrase_id\"", req.ID),
+		)
+		return
+	}
+
+	resp.Diagnostics.Append(resp.State.SetAttribute(ctx, path.Root("environment_id"), attributes[0])...)
+	resp.Diagnostics.Append(resp.State.SetAttribute(ctx, path.Root("id"), attributes[1])...)
+}
+
+func (p *voicePhraseResourceModel) expand() (*verify.VoicePhrase, diag.Diagnostics) {
+	var diags diag.Diagnostics
+
+	data := verify.NewVoicePhrase(p.DisplayName.ValueString())
+
+	data.SetId(p.Id.ValueString())
+
+	if !p.CreatedAt.IsNull() && !p.CreatedAt.IsUnknown() {
```

Comment:
`CreatedAt` and `UpdatedAt` shouldn't be handled as input parameters in the `expand` method